Units/simple-elixir.d/input.ex

```diff
@@ -1,18 +1,190 @@
+# Most of this code is taken from the official Elixir documentation.
+# With comments and modifications
+# by Ivan Gonzalez Polanco <ivan14polanco@gmail.com>
 #
-# Taken from https://elixir-lang.org/
+# This code doesn't compile, since it's supposed to test de ctags elixir
+# parser, not the elixir elixir parser.
+
+#
+# f functions (def ...)
+#
+defmodule MyString do
```

Comment:
Just a question. defmodule can be nested?
If not, you can build a scope. I will make more advises later.

Reply:
See a test case in #2025. It is an example of how scope flags work.

Reply:
Yes, defmodule can be nested.